lib/components/Insights/Charts/VBarChartInsight/index.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import BarChartStory from "@/components/Charts/VerticalBarChart/index.stories"
+import { VerticalBarChartInsight } from "."
+import { containerStoryArgs } from "../storybook-utils"
+
+const meta = {
```

Comment:
You're gonna kill me for this, but the folder name is still the old one (VBarChartInsight) 👀